pip_requirements.txt

```diff
@@ -4,7 +4,7 @@
 
 # Pattoo packages
 
-PattooShared==0.0.87
+PattooShared==0.0.89
```

Comment:
[This may have to be version 90. ](https://pypi.org/project/PattooShared/#history)

Reply:
this causes the build to fail since the highest version of pattoo shared is currently 89.